books.json

```diff
@@ -52,6 +52,18 @@
             "author": "Stinson, Kathy",
             "faylib": "http://catalog.faylib.org/polaris/search/title.aspx?ctx=3.1033.0.0.1&cn=529040",
             "isbn": "9781554519002"
+        },
+        {
+            "title": "Luke & the little seed",
+            "author": "Ferri, Giuliano",
+            "faylib": "http://catalog.faylib.org/polaris/search/title.aspx?ctx=1.1033.0.0.1&pos=3",
```

Comment:
dead link 😕 it seems to be tied to your session.

you can just remove this. I'm considering removing the links from the doc entirely.